### ISSUE TITLE
Reset saved data (stretch goal)

### ISSUE THREAD

**Issue by MattPahuta** [MattPahuta/salesboard-app_scrimba-gp#7]
Add reset of saved (local storage) data - button on HTML and associated JS code.

Important: Remember to first [pull down](https://scrimba.com/learn/frontend/pulling-cvKk6PU8) all the latest changes from the remote repository.

Instructions: [Create a new branch in your repository](https://scrimba.com/learn/frontend/creating-a-branch-c4zBakHg) for this task. In this new branch, you’ll update the project by creating a “reset” button to let the user reset the Salesboard data.

When complete, commit your changes to your feature branch and publish it to the remote repo. Then, [create a pull request](https://scrimba.com/learn/frontend/creating-a-pull-request-cz9nJ9sZ) and request someone in your group to review it. Once the PR is approved, merge the changes in your pull request into the repo’s main branch.

**Comment by MattPahuta** [comment]
Enhancement complete. Ability to reset data saved in local storage along with corresponding markup and styling has been implemented and merged with main branch.
